Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Compact/Triginometry/CompactArctangentNodeable.cpp

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "CompactArctangentNodeable.h"
+
+namespace ScriptCanvas::Nodeables
+{
+    float CompactArctangentNodeable::In(float a)
+    {
+        return atan(a);
```

Comment:
any atan2 node plans for the future?

Reply:
I think Atan2 is probably the more common one that is used, because it avoids the issue with determining the angle when the x-axis is 0. 
i.e is the angle -pi/2 or pi/2 ?

Reply:
In that case, I can go ahead and add it here